docs/source/recipes/testing-libraries.md

```diff
@@ -16,3 +16,28 @@ module.exports = ['@testing-library/react/cleanup-after-each', '@testing-library
 ```
 
 More Info on Jest `setUpFilesAfterEnv` [here](https://jestjs.io/docs/en/configuration#setupfilesafterenv-array)
```

Comment:
Should be `setupFilesAfterEnv` but everything else LGTM

Reply:
good catch 👍